components/profileImageComponents/ProfileImageAndNameWithFriend.tsx

```diff
@@ -83,11 +76,11 @@ const ProfileImageAndNameWithFriend = ({
               >
                 {renderFriendStatus(friendStatus)}
               </Headline>
-              <FriendToast
+              <ToastWithIcon
                 requestSent={requestSent}
                 setRequestSent={setRequestSent}
-                friendStatus={friendStatus}
-                offset={toastOffset}
+                isVisible={friendStatus === "friend-request-pending"}
```

Comment:
More readable 👍

 Double-check the possible issue of the toast appearing if the status starts out as "friend-request-pending".